electives/18623.md

```diff
@@ -0,0 +1,66 @@
+
+You will have a few short lab sessions that will catch you up on the basics of Cadence simulation and layout. But you'll really start to become familiar with its tools through working on the projects.
+Note that the projects are done over roughly the course of one month each. This may sound like a lot of time, but you will also have homeworks and exams in the course during this time - the earlier you start on the projects, the better!
+
+### Project 1: Transimpedance Amplifier (TIA)
+
+This project has you build a transimpedance amplifier, which is a current-to-voltage converter. The TIA is broken into three stages: an input stage to convert the input current to a voltage, a gain stage to amplify the voltage, and an output stage to provide a buffer for the output signal. You will also create biasing circuitry for the TIA, as well as testbenches to test each part of the circuit.
+
+Interestingly enough, you may find the first project to be much more difficult than the second project. This is because you are likely still going to be learning the basics of Cadence, as well as developing your understanding of amplifier topologies.
+Also, although you are given a basic topology for the TIA design, you will need to revise the design significantly in order to meet the desired specifications.
+
+That said, this project lends itself well to a divide-and-conquer approach. You and your teammate can decide who takes each component of the TIA. Then, you can slowly put the pieces together.
+Before you begin your work in Cadence, though, be sure to spend a large amount of time on the design. You should work out the different characteristics of your circuit before implementing it in Cadence. Use the design review to get good feedback from the professor and TAs.
+
+### Project 2: Operational Amplifier (Op-Amp)
+
+This project has you build an op-amp, which you have likely used as a building block in a larger circuit from earlier electronics classes. The op-amp is broken into two stages: a differential pair stage and a high-gain amplifier stage. Like the previous project, you will also create biasing circuitry for the op-amp; testbenches are also a good idea.
+
+You may find this project to be a bit easier than the previous one. This is because by this point, you have learned the tools of Cadence.
+Additionally, some of the lectures from the middle of the class actually step through how to design an op-amp to meet certain specifications. You should definitely use these slides as an aid in designing your own op-amp for this project.
+
+Similar to the previous project, work with your teammate to split up portions of the design, test them individually, and then integrate together. You will also need to do layout for this project, but that should be saved for after you have finalized your simulations.
+
```

Comment:
So I was not a great student in this class, but one thing I remember was that a lot of labs just felt like randomly guessing parameters / topologies and hoping it would work. And sometimes you would just go to OH and the TA would just magically come up with a topology for you to try and get to work.

Is there a better way to come up with / design the circuits, or is it really just guessing? Curious if you have any insight on this.

Reply:
I struggled a lot on the projects :D so unfortunately I don't think I have the best advice besides telling folks to go to office hours to discuss their designs with the TAs. At least from my experience, the TAs were very helpful in walking me through my design and pointing me in the right direction. Some of the designs still seemed a bit mysterious (particularly for project 1) but OH definitely helped! I added a bit to talk about that